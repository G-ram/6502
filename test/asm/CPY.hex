a9 08 a0 08 8d 00 01 cc 00 01
