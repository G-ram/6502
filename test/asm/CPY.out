A: 08
X: 00
Y: 08
P: 00010011
S: 00
PC: 60a
100 : 08
101 : 00
